samples/CloudClipboard/CloudClipboard/CloudClipboard/wwwroot/js/site.js

```diff
@@ -0,0 +1,4 @@
+﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
```

Comment:
We should remove this if there's nothing in here.   Our goal is to provide a sample of how to use the SDK;  this doesn't contribute to that goal.

Reply:
Everything in wwwroot is part of the ASP.NET template.  I wouldn't bother cleaning it up.

Reply:
Did not clean it up